Modulo9-BrunoMonteiro/src/exercicio13/MapSet.java

```diff
@@ -0,0 +1,51 @@
+package exercicio13;
+
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class MapSet {
+
+	public static void main(String[] args) {
+		HashMap<Integer, Aluno> hashmap = new HashMap<Integer, Aluno>();
+		HashSet<Aluno> hashset = new HashSet<Aluno>();
+
+		inserirAlunosHashMap(hashmap);
+		inserirAlunosHashSet(hashset);
+		
+		long inicio = System.currentTimeMillis();
+		for (int i = 0; i < 3000000; i++) { // fiz esse for porque s� uma vez sempre � 0ms
```

Comment:
O for para conseguir imprimir os segundos não ficou muito legal, mas é comprendido o seu próposito.